crates/holochain/src/core/workflow/app_validation_workflow.rs

```diff
@@ -59,7 +59,7 @@ mod types;
 pub async fn app_validation_workflow(
     dna_hash: Arc<DnaHash>,
     workspace: Arc<AppValidationWorkspace>,
-    fetched_dependencies: Arc<Mutex<HashSet<AnyDhtHash>>>,
+    validation_dependencies: Arc<Mutex<HashSet<AnyDhtHash>>>,
```

Comment:
Seems kind of incongruous that we also have `validation_dependencies: ValidationDependencies` in sys validation with a different struct. The ship has sailed but I'm wondering why there wasn't an opportunity for code re-use there? Is there an intention to eventually unify them?

Reply:
I took inspiration from the sys validation TestCase, but there wasn't much overlap between members of the two. If in the end of the refactor it turns out to have enough overlap, I'd unify them.